runtime/src/iouring/mod.rs

```diff
@@ -17,6 +25,15 @@ pub struct Config {
     pub iopoll: bool,
     /// If true, use single issuer mode.
     pub single_issuer: bool,
+    /// In the io_uring event loop, wait at most this long for a new completion
+    /// before checking for new work to submit to the ring.
+    /// If None, wait indefinitely. In this case, caller must ensure that operations
+    /// submitted to the io_uring complete so as to not block the event loop.
+    /// For example, do not submit a recv operation and then a write operation
+    /// that satisfies the recv operation; the event loop may block while
+    /// waiting for the recv operation to complete, and never submit the write,
+    /// causing a deadlock.
+    pub poll_new_work_freq: Option<Duration>,
```

Comment:
Not crazy about this name...open to suggestions

Reply:
I suggest `force_poll` (you may want to change `iopoll` to `io_poll` for consistency)?

Reply:
I don't think this should be an `enum` with `iopoll` (as you may want to do both)?

_This is what you currently have, just adding more color._

Reply:
I think 2 Options makes more sense than an enum